redpanda/templates/poddisruptionbudget.yaml

```
@@ -35,3 +36,4 @@ spec:
       app.kubernetes.io/name: {{ template "redpanda.name" . }}
       app.kubernetes.io/instance: {{ .Release.Name | quote }}
   maxUnavailable: {{ .Values.statefulset.budget.maxUnavailable | int64 }}
+{{- end }}
```

> For our current use case there is no HA requirement, we plan to use _Redpanda_ only to create ephemeral one-node ‘clusters’ that are used in component testing.

In that case, we should probably add a value under "below here be dragons" with a big warning about not using in production, then use that new value in an else, here.

something like:

```
yesyesyesno:
  pod_disruption_budget:
    enabled: true
```    

```suggestion
{{- else }}
  {{- if .Values.yesyesyesno.pod_disruption_budget.enabled }}
  fail "You must use Kubernetes 1.21+ for a production cluster. If you're not using this in production and you know why doing this is a bad idea, you can set yesyesyesno.pod_disruption_budget.enabled=false to continue."
  {{- end }
{{- end }}
```

(The parent key is not what I suggest, it's just a humorous placeholder)

---

I committed largely what you propose above, including an attempt for a more serious key name 😄